<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>等线</family>
         </font>
        </property>
        <property name="text">
         <string>命令名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cmdName">
        <property name="font">
         <font>
          <family>等线</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>等线</family>
         </font>
        </property>
        <property name="text">
         <string>命令语句：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="cmdContent">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>等线</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="cmdId">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>134</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
    </font>
   </property>
   <property name="text">
    <string>cmd1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
